src/hooks/useQuizState/useQuizApiState.ts

```diff
@@ -40,28 +40,30 @@ const useQuizApiState: UseQuizApiState = (
   const { quizId, quizVersionId: quizVersionIdProp, resultsPageOptions } = quizOptions;
   const { queryItems, queryAttributes, isSharedResultsQuery } = useQueryParams();
   const dispatchQuizResults = async () => {
-    try {
-      const quizResults = await getQuizResults(cioClient, quizId, {
-        answers: quizLocalState.answers,
-        resultsPerPage: resultsPageOptions?.numResultsToDisplay,
-        quizVersionId: quizLocalState.quizVersionId,
-        quizSessionId: quizLocalState.quizSessionId,
-      });
-      // Set quiz results state
-      dispatchApiState({
-        type: QuizAPIActionTypes.SET_QUIZ_RESULTS,
-        payload: {
-          quizResults,
-        },
-      });
-      if (!quizLocalState.isQuizCompleted)
-        dispatchLocalState({
-          type: QuestionTypes.Complete,
+    if (quizLocalState.answers.length) {
+      try {
+        const quizResults = await getQuizResults(cioClient, quizId, {
+          answers: quizLocalState.answers,
+          resultsPerPage: resultsPageOptions?.numResultsToDisplay,
+          quizVersionId: quizLocalState.quizVersionId,
+          quizSessionId: quizLocalState.quizSessionId,
         });
-    } catch (error) {
-      dispatchApiState({
-        type: QuizAPIActionTypes.SET_IS_ERROR,
-      });
+        // Set quiz results state
+        dispatchApiState({
+          type: QuizAPIActionTypes.SET_QUIZ_RESULTS,
+          payload: {
+            quizResults,
+          },
+        });
+        if (!quizLocalState.isQuizCompleted)
+          dispatchLocalState({
+            type: QuestionTypes.Complete,
+          });
+      } catch (error) {
```

Comment:
To test remove `if(quizLocalState.answers.length)` and add a debugger here. You'll see that for one of the rerenders there's an error for "answers required"

When we add a UI for the error state this will show an error message for a split second

The "if" prevents that